.github/tests/forks/fork11.yml

```diff
@@ -1,6 +1,7 @@
 args:
-  zkevm_rollup_fork_id: 11
```

Comment:
do we plan to switch to non-personal repo images in the near future ?

Reply:
I'd like too yes